### PR TITLE
Project and Swagger-UI setup.

### PR DESCRIPTION
**Pull request by cabol**
Project and Swagger-UI setup.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+SHELL_DEPS = sync
+
+dep_sync =  git https://github.com/inaka/sync.git 0.1.3
+
+TEST_DEPS = xref_runner
+
+dep_xref_runner = git https://github.com/inaka/xref_runner.git 0.2.2
+
+PLT_APPS := cowboy
```
Comment (elbrujohalcon) [inline Makefile]: you should add `trails` here

**Comment by gadgetci** [comment]
According to **Dialyzer**:
> 
dialyzer: Analysis failed with error:
No .beam files to analyze (no --src specified?)

File: src/cowboy_swagger.app.src [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{application, cowboy_swagger,
+ [
+  {description,
+   "Swagger for Cowboy Erlang projects"},
+  {vsn, "0.0.1"},
+  {applications,
+   [kernel,
+    stdlib
```
Comment (elbrujohalcon) [inline src/cowboy_swagger.app.src]: trails should be here, although it's inconsequential since this is no real app.

**Comment by elbrujohalcon** [comment]
@cabol please make sure that both `make dialyze` and `make test` work. At least add [`xref_SUITE`](http://inaka.net/blog/2015/07/17/erlang-meta-test/) in your test folder

**Pull request merged**